### PR TITLE
📼 Intro video & logo

### PR DESCRIPTION
**Pull request by jakubfiala**
This PR:

+ adds favicon and various meta icons for mobile browsers etc.
+ adds HTML & styles for the intro video and logo part of the site
+ makes it (kinda) work across breakpoints and on IE9

screen shot 2017-08-29 at 11 36 41
screen shot 2017-08-29 at 11 36 53

### CODE REVIEW

File: components/header/_header.scss [hunk 1]
```diff
@@ -30,23 +53,15 @@
...
-
-  .main-nav--desktop__item--last {
-    @include rem(margin-left, spacing(s));
+    width: 55%;
   }
 }
 
 @include respond-to(breakpoint(l)) {
   .main-nav--desktop__list {
     padding: spacing(xl) spacing(l) spacing(xl) 0;
```
Comment (RhodesPeter) [inline components/header/_header.scss]: Can this be inside an @include?

Comment (RhodesPeter) [inline components/header/_header.scss]: Can this be inside an @include?

File: components/header/_header.scss [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+$logo-spacing-top-s: spacing(xxl) + spacing(s);
+
+.logo-wrapper {
+  left: 0;
+  padding: $logo-spacing-top-s $body-padding 0;
+  position: absolute;
+  top: 0;
+
+  @include respond-to(breakpoint(s)) {
+    padding-top: spacing(s);
```
Comment (RhodesPeter) [inline components/header/_header.scss]: Inside an @include?

File: components/header/_header.scss (hunk 1 above)
Comment (jakubfiala) [reply components/header/_header.scss]: sure!

File: components/header/_header.scss (hunk 2 above)
Comment (jakubfiala) [reply components/header/_header.scss]: yep, should be

**Pull request merged**